apps/dot-voting/app/components/VoteDetails/index.js

```diff
@@ -54,8 +61,28 @@ const VoteDetails = ({ app, vote, userAccount, onVote }) => {
       }
     }
 
+    function loadUserBalance() {
+      const { snapshotBlock } = vote.data
+      if (tokenContract && userAccount) {
+        combineLatest(
+          tokenContract.balanceOfAt(userAccount, snapshotBlock),
+          tokenContract.decimals(),
+          tokenContract.symbol()
+        )
+          .pipe(first())
+          .subscribe(([ balance, decimals, symbol ]) => {
+            setTokenData({
+              userBalance: balance,
+              decimals: decimals,
+              voteTokenSymbol: symbol,
```

Comment:
I don't think we ever use this value, `voteTokenSymbol`. If that's true, we can stop importing `tokenSymbolAbi` altogether.

Reply:
Ok. It is useful in store only it seems. Removed.